exchanges/subscription/list.go

```diff
@@ -169,3 +156,17 @@ func (l List) Public() List {
 	}
 	return n
 }
+
+// populate adds all enabled pairs for an asset to the assetPair map
+func (ap assetPairs) populate(a asset.Item, e IExchange) error {
```

Comment:
suggestion: can swap the params around for alignment to calls.

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/4bf3d04bc